enable_button: 1
stop_button: 2
e_stop_button: 3
linear_speed_axis: 1
angular_speed_axis: 0

max_linear_speed: 1.0
max_angular_speed: 1.0

enable_e_stop: 1

cmd_vel_topic: joy/cmd_vel
e_stop_pub_topic: e_stop_sw_enable

joy_topic: joy
